sys/fido2/ctap/transport/hid/Kconfig

```diff
@@ -4,13 +4,17 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-menuconfig KCONFIG_USEMODULE_FIDO2_CTAP_TRANSPORT_HID
+menuconfig MODULE_FIDO2_CTAP_TRANSPORT_HID
     bool "FIDO2 CTAP TRANSPORT HID"
-    depends on USEMODULE_FIDO2_CTAP_TRANSPORT_HID
+    select MODULE_ISRPIPE
+    select MODULE_USBUS
```

Comment:
I'm thinking that it probably would make sense to depend on USBUS to be there, as currently this is not depending on actually having `PERIPH_USBDEV` or `MOCK_USBDEV`.

Reply:
Makes sense to me. Then the only way for it to work would be to explicitly use the USBUS module, right ?
E.g for `app.config.test` this would mean: 
```txt
CONFIG_MODULE_AUTO_INIT_USBUS=n
CONFIG_MODULE_USBUS=y
CONFIG_MODULE_FIDO2_CTAP=y
CONFIG_MODULE_FIDO2_CTAP_TRANSPORT=y
CONFIG_MODULE_FIDO2_CTAP_TRANSPORT_HID=y
```

Reply:
Yes, but I don't find it too bad to enable USB when you know you are using FIDO.